crates/extensions/tedge_log_manager/src/actor.rs

```diff
@@ -0,0 +1,269 @@
+use async_trait::async_trait;
+use http::status::StatusCode;
+use log::error;
+use log::info;
+use log_manager::LogPluginConfig;
+use serde_json::json;
+use tedge_actors::fan_in_message_type;
+use tedge_actors::Actor;
+use tedge_actors::ClientMessageBox;
+use tedge_actors::LoggingSender;
+use tedge_actors::MessageReceiver;
+use tedge_actors::NoMessage;
+use tedge_actors::RuntimeError;
+use tedge_actors::Sender;
+use tedge_actors::SimpleMessageBox;
+use tedge_file_system_ext::FsWatchEvent;
+use tedge_http_ext::HttpError;
+use tedge_http_ext::HttpRequest;
+use tedge_http_ext::HttpRequestBuilder;
+use tedge_http_ext::HttpResponseExt;
+use tedge_http_ext::HttpResult;
+use tedge_mqtt_ext::MqttMessage;
+use tedge_mqtt_ext::Topic;
+use tedge_utils::paths::PathsError;
+
+use super::error::LogManagementError;
+use super::json::CommandStatus;
+use super::json::LogRequestPayload;
+use super::json::LogResponsePayload;
+use super::LogManagerConfig;
+
+fan_in_message_type!(LogInput[MqttMessage, FsWatchEvent] : Debug);
+fan_in_message_type!(LogOutput[MqttMessage]: Debug);
+
+pub struct LogManagerActor {
+    config: LogManagerConfig,
+    pub plugin_config: LogPluginConfig,
```

Comment:
You don't need `pub`.

```suggestion
    plugin_config: LogPluginConfig,
```